extension/src/popup/views/AddAccount/ImportAccount/index.tsx

```diff
@@ -53,20 +55,23 @@ export const ImportAccount = () => {
 
   return (
     <div className="ImportAccount">
+      <SubviewHeader title="Import Stellar Secret Key" />
       <div className="ImportAccount__warning-block">
         <InfoBlock variant={InfoBlock.variant.warning}>
-          <div className="ImportAccount__warning-header">
-            read before importing your key
-          </div>
-          <div className="ImportAccount__warning-copy">
-            <p>
-              Freighter can't recover your imported secret key using your backup
-              phrase. Storing your secret key is your responsability.{" "}
-            </p>
-            <p>
-              Freighter will never ask for your secret key outside of the
-              extension.
-            </p>
+          <div>
+            <div className="ImportAccount__warning-header">
+              read before importing your key
+            </div>
+            <div className="ImportAccount__warning-copy">
+              <p>
+                Freighter can’t recover your imported secret key using your
+                backup phrase. Storing your secret key is your responsability.{" "}
```

Comment:
fairly certain this was originally me since I have a google history search for "responsability" 😆 
```suggestion
                backup phrase. Storing your secret key is your responsibility.{" "}
```

Reply:
😬